### ISSUE TITLE
Exception raised when using Gretel in multithreaded environment

### ISSUE THREAD

**Issue by MrShemek** [kzkn/gretel#13]
We have a table with thousand cities for a city directory and for each of them we instantiate a crum in `config/breadcrumbs.rb` and also we have some other crumbs like this

```ruby
crumb :help_cancel_tickets do
    link t("help.menu.help_cancel_tickets"), help_cancel_tickets_path
    parent :help
  end
```

Then we call it in the view:

```ruby
<% breadcrumb :help_cancel_tickets %>

...
#in the layout
<%= breadcrumbs separator: " &rsaquo; " %>
```

We were using Puma with 4 workers and 4 threads and sometimes we had
`ActionView::Template::Error: Breadcrumb :help_cancel_tickets not found.` error

But now we increased to 16 threads and errors like this are much more common, why does this happen? I presume 2 possible things:

* all crumbs are initialized in the first thread so when a request lands in another thread, they dont exist in the thread memory?
* Each thread creates in memory their own set of crumbs configured in breadcrumbs.rb but because there are so many, sometimes a request can be processed before all crumbs are in memory?

any thoughts?

---

Original issue: [#104](https://github.com/lassebunk/gretel/issues/104)

**Comment by kzkn** [comment]
@silva96
I've pushed the fix to master.  The exception has no longer occurred in my minimal environment for reproduction the problem..
If you can, give it try in your environment.

**Comment by kzkn** [comment]
released in 4.1.0
